sunxspex/emission.py

```diff
@@ -348,6 +444,172 @@ def brm_guass_legendre(x1, x2, npoints):
     return x, w
 
 
+def brm2_dmlin(a, b, maxfcn, rerr, eph, eelow, eebrk, eehigh, p, q, z, efd=True, verbose=False):
```

Comment:
This and `brm2_dmlino` are very similar and could be generalised into one function - bit of work so prob worth discussing if its worth it or not.

Reply:
Agreed. There is perhaps a way to merge them, but it would take some thinking.